controllers/productController.js

```diff
@@ -0,0 +1,12 @@
+const productService = require("../services/productService");
+const { catchAsync } = require("../utils/error");
+
+const getProductsByParameter = catchAsync(async (req, res) => {
+  const products = await productService.getProductsByParameter(req.query);
+
+  return res.status(201).json(products);
```

Comment:
```suggestion
  return res.status(200).json(products);
```
`201`은 레코드 생성을 의미합니다. 다른 상태코드로 변경해 주세요~